incubator/hnc/pkg/testutils/testutils.go

```diff
@@ -216,6 +216,7 @@ func CleanupNamespaces(nses ...string) {
 	for _, ns := range nses {
 		TryRunQuietly("kubectl hns set", ns, "-a")
 		TryRunQuietly("kubectl annotate ns", ns, "hnc.x-k8s.io/subnamespaceOf-")
+		TryRunQuietly("kubectl annotate ns", ns, "hnc.x-k8s.io/subnamespace-of-")
```

Comment:
Why are we keeping the old annotation if (as of this change) it should never exist or do anything meaningful?

Reply:
Also, can you please update the similar code I added in the rolebinding test?

Reply:
Oh this function cleans up namespaces in both v0.5 and v0.6. Updating using your RB test code sgtm.

Reply:
Hmm, maybe comment why we still need to clean up the v0.5 versions and when this can be removed?

Reply:
Updated the code and comment.

Reply:
I'm not seeing the rolebinding code updated in the latest commit?

Reply:
What rolebinding code you are talking about? I thought it's https://github.com/kubernetes-sigs/multi-tenancy/pull/1164/commits/6665340e7956fccc73e98212447c6f91301174ec that we will have `TryRunQuietly()` to make sure the namespace exists and use `MustRunWithTimeout()`. is it not what you want?

Reply:
Resolved offline, my change is in testutils not rolebinding, and Yiqi had to rebase to see it (but there was no line conflict so Github didn't alert her that she needed to rebase).

Reply:
(the change is in `RecoverHNC()`)

Reply:
Updated and tested rolebinding with `FIt` in `make test-e2e`.